Documentation/ceph-common-issues.md

```diff
@@ -201,13 +220,21 @@ If all the preceding sections have been successful or inconclusive, then take a
 You can determine which `rook-ceph-agent` is running on the same node that your pod is scheduled on by using the `-o wide` output, then you can get the logs for that `rook-ceph-agent` pod similar to the example below:
 
 ```console
-> kubectl -n rook-ceph get pod -o wide
+kubectl -n rook-ceph get pod -o wide
+```
+
+```console
 NAME                                  READY     STATUS    RESTARTS   AGE       IP             NODE
 rook-ceph-agent-h6scx                 1/1       Running   0          9m        172.17.8.102   172.17.8.102
 rook-ceph-agent-mp7tn                 1/1       Running   0          9m        172.17.8.101   172.17.8.101
 rook-operator-2203999069-3tb68        1/1       Running   0          9m        10.32.0.7      172.17.8.101
+```
+
+```console
+kubectl -n rook-ceph logs rook-ceph-agent-h6scx
```

Comment:
Since it's a specific pod name, we don't expect it to be copied so we can keep the `$` and a single block with the output, right?